Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -436,6 +436,8 @@ for this property.
 - (void)_convertPoint:(CGPoint)point fromFrame:(WKFrameInfo *)frame toMainFrameCoordinates:(void (^)(CGPoint, NSError *error))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 - (void)_convertRect:(CGRect)rect fromFrame:(WKFrameInfo *)frame toMainFrameCoordinates:(void (^)(CGRect, NSError *error))completionHandler WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
 
++ (NSString *)_userVisibleStringForURL:(NSURL *)url WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA));
```

Comment:
can this ever return nil? if so should it be marked nullable?

Reply:
If I added nullability annotations for this method, I'd have to add them for all methods in this header.

Reply:
maybe we could add NS_ASSUME_NONNULL_BEGIN / NS_ASSUME_NONNULL_END to this file and then it would be fine to only add the nullability annotation to this method?

Reply:
That would make it start assuming that everything without annotation is non-null, which is certainly incorrect.  I would have to go through all the selectors and annotate them, which is why it hasn't been done yet.